src/py/flwr/client/supernode/app.py

```diff
@@ -119,6 +119,17 @@ def run_client_app() -> None:
     register_exit_handlers(event_type=EventType.RUN_CLIENT_APP_LEAVE)
 
 
+def exec_client_app() -> None:
```

Comment:
As we use `exec` for the SuperExec I would prefer to reuse that word on the SuperNode side and just change all mentions of `exec_` to `start_` as in `start_client_app`

Reply:
Updated to `flwr_clientapp`, as discussed.